### PR TITLE
ida-free@7.0: Fix installation

### LINKED ISSUE #3183: ida-free@7.0: decompress error
# Command line output
```
Extracting idafree70_windows.exe ... ERROR Exit code was 1!
Failed to extract files from C:\Users\****\scoop\apps\ida-free\7.0\idafree70_windows.exe.
Log file:
  ~\scoop\apps\ida-free\7.0\innounp.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/lukesampson/scoop-extras/issues/new?title=ida-free%407.0%3a+decompress+error
```
image

# innounp.log
```
The setup files are corrupted or made by incompatible version. Maybe it's not an Inno Setup installation at all.
(00491A8A)
```
# More
- It is an installation exe. I think that is why the innounp process throw exception.
- Maybe the ida change their exe to installation one instead of the previous zip one if the previous versions worked.

### PR DESCRIPTION
**Pull request by xycui**
- Closes #3183

### CODE REVIEW

**Comment by Ash258** [comment]
Unacceptable from my side due to elevated privileges requirement. At least add pre_install block with check for admin status of user and failsafe from installation if no.

**Comment by xycui** [comment]
> Unacceptable from my side due to elevated privileges requirement. At least add pre_install block with check for admin status of user and failsafe from installation if no.

Make sense

File: bucket/ida-free.json [hunk 1]
```diff
@@ -6,11 +6,18 @@
         "url": "https://www.hex-rays.com/products/ida/licenses.shtml"
     },
     "version": "7.0",
-    "innosetup": true,
     "architecture": {
         "64bit": {
             "url": "https://out7.hex-rays.com/files/idafree70_windows.exe",
             "hash": "sha1:38b41baf3dd60dba67a7e656c5e5beb2d4a0adca",
+            "installer": {
```
Comment (Ash258) [inline bucket/ida-free.json]:
```suggestion
            "pre_install": "if (-not (is_admin)) { error 'Administrator privileges are needed'; break }",
            "installer": {
```

**Comment by Ash258** [comment]
Where is uninstaller?

**Comment by xycui** [comment]
> Where is uninstaller?

@Ash258 I have updated the uninstall script. I am still the newbie to scoop installer. Thanks for your patient. 

screen_record

**Pull request merged**